doc/styles/Vocab/ANSYS/accept.txt

```diff
@@ -2,4 +2,7 @@ ANSYS
 Ansys
 ansys
 OpenAPI
-OpenAPI-Common
\ No newline at end of file
+OpenAPI-Common
+Kerberos
+Heimdal
+APIs
```

Comment:
I wasn't sure about this, but Google seem to use it in their API documentation https://cloud.google.com/apis/design/resources